core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -213,6 +216,10 @@ trait SubmissionComponent {
         }
     }
 
+    def updateSubmissionUserComment(submissionId: UUID, newComment: String): ReadWriteAction[Int] = {
```

Comment:
Why is the return value ReadWriteAction[Int]?

Reply:
`ReadWriteAction[Int]` is type alias of [DBIOAction](https://scala-slick.org/doc/3.2.0/dbio.html) that has Read and Write effect.